documentation/milestones/individual_milestones/get_opportunities.md

```diff
@@ -0,0 +1,196 @@
+- As a **consumer of the API**, I want:
+  - clear documentation and a user guide for the API, so that I don't have to rely on reading the source code to learn how to interact with and consume from it
+  - changes made to a given endpoint to be backward-compatible, so that I can start building against this API without worrying about breaking changes
+- As a **project maintainer**, I want:
+  - most of the critical development infrastructure to be in place when we officially launch the API, so that we can deploy bug fixes or new features quickly once the site is live
+- As an **open source contributor**, I want:
+  - to be able run the code for the API locally, so that I can test my contributions to the codebase
+
+## Technical description
+
+### Infrastructural Requirements
+
+The infrastructure developed to deploy and host the API should balance:
+
+- Code quality
+- Security
+- Delivery velocity
+- Cost & maintenance
+
+### Developer Experience Requirements
+
+The API should be developed and versioned in a way that balances:
+
+- Discoverability
+- Ease of adoption
+- Backwards compatibility
+
+### Data Requirements
+
+The way that data is stored and delivered through the API should balance:
+
+- Eventual consistency with legacy Grants.gov
+- Improvements to the existing data model
+- Ease of managing schema changes
+
+### Definition of done
+<!-- Required -->
+
+- [ ] The following infrastructure requirements are satisfied:
+  - [ ] The code needed to build and deploy the site is merged to `main`
+  - [ ] The site is built and hosted with the tools selected in the [API Planning](https://github.com/HHS/grants-equity/issues/42) and [DB Planning](https://github.com/HHS/grants-equity/issues/48) milestones
+  - [ ] All code quality checks set up in the [Developer Tools milestone](https://github.com/HHS/grants-equity/issues/50) are passing
+  - [ ] The resources required to deploy and host the API are provisioned programmatically using the [Infrastructure-as-Code milestone](https://github.com/HHS/grants-equity/issues/123) framework
+  - [ ] Code changes are deployed using the CI/CD pipeline set up in [the Back-end CI/CD milestone](https://github.com/HHS/grants-equity/issues/57)
+  - [ ] Logging/monitoring is configured, and it both records the metrics defined below and alerts the development team when the API is down
+- [ ] The following developer experience (DX) requirements are satisfied:
+  - [ ] The API is live at `beta.grants.gov/api/`
+  - [ ] Developers can learn how to interact with the API by referencing the API documentation
+  - [ ] The endpoint path indicates which major version of the API the developer is consuming
+  - [ ] Breaking changes are reflected in predictable changes to the endpoint path
+  - [ ] The endpoint is available when legacy grants.gov experiences planned or unplanned downtime, maintenance, and upgrades
```

Comment:
Do we want to add these as necessary conditions? 

- [ ] Text fixture data is scripted to provide consistent and reliable test fixtures for integration tests and local development
- [ ] All developers (including open source contributors) are able to spin up a database replica including test fixture data so that they can conduct local development.
- [ ] Feature flag framework is implemented so that functionality can be deployed to PROD without being turned on

Reply:
Yeah I like these additions, added in [this commit](https://github.com/HHS/grants-equity/pull/249/commits/3aa14fa418605012f4e9766952271ca22b97e8f2)